client/login/wp-login/test/test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External Dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal Dependencies
+ */
+import { Login } from '../index.jsx';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+const defaultProps = {
```

Comment:
I think that's fine, since these are the parameters `Login` component depends on.